themes/bootstrap3/templates/search/results.phtml

```diff
@@ -33,7 +33,7 @@
   }
 
   // Enable cart if appropriate:
-  $this->showCartControls = $this->params->getOptions()->supportsCart() && $this->cart()->isActive();
+  $this->showCartControls = $this->params->getOptions()->supportsCart() && $this->cart()->isActive() && $this->showBulkOptions;
```

Comment:
With this check in place, we only get cart controls and checkboxes when showBulkOptions is true, regardless of whether or not bookbagTogglesInSearch is turned on. That means there is no way to get the traditional cart behavior without also enabling other bulk operations. I think you need to take the additional setting into consideration.